versions.lock

```diff
@@ -60,14 +61,14 @@ io.dropwizard.metrics:metrics-jmx:4.1.5 (1 constraints: 0c050736)
 io.dropwizard.metrics:metrics-jvm:4.1.5 (1 constraints: 0c050736)
 io.jaegertracing:jaeger-core:1.1.0 (1 constraints: 5c0d6c26)
 io.jaegertracing:jaeger-thrift:1.1.0 (1 constraints: 0405f335)
-io.netty:netty-buffer:4.1.47.Final (6 constraints: 29558e4c)
-io.netty:netty-codec:4.1.47.Final (2 constraints: fc135b82)
-io.netty:netty-common:4.1.47.Final (8 constraints: 9d6ed3cf)
-io.netty:netty-handler:4.1.47.Final (2 constraints: 2f176f42)
-io.netty:netty-resolver:4.1.47.Final (3 constraints: ad213fd0)
-io.netty:netty-transport:4.1.47.Final (5 constraints: 7847722a)
-io.netty:netty-transport-native-epoll:4.1.47.Final (2 constraints: 2f176f42)
-io.netty:netty-transport-native-unix-common:4.1.47.Final (2 constraints: 081ad205)
+io.netty:netty-buffer:4.1.50.Final (6 constraints: 0555463a)
```

Comment:
See here? There are 6 other dependencies that include this one. That's where version consolidation conflicted with ant, probably.

Reply:
OK, thanks. Nothing else to do here, though I take it.